PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,10 @@
 #### What does this PR do?
 #### Who is reviewing it? 
-(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
+<!--(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)-->
 #### Select relevant component teams: 
 https://github.com/orgs/codice/teams
 #### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
-(please choose ONLY two committers from below, delete the rest)
+<!--(please choose ONLY two committers from below, delete the rest)-->
```

Comment:
✏️ How about commenting out these too?

- `If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers.`
- `(List steps with links to updated documentation)`
- `(if appropriate)`